src/calibrate/Dockerfile

```diff
@@ -1,6 +1,9 @@
 # Inherit from the Python Docker image
 FROM python:3.7-slim
 
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install tk
```

Comment:
Are we still using non-docker setup? I think this should be documented somewhere too (like in the readme) for cases where we are not using docker. Just thinking here 🤔 .

Reply:
Here, we are actually using Docker @ja-odur